.github/workflows/ci.yaml

```diff
@@ -43,16 +43,3 @@ jobs:
           version: ">= 363.0.0"
       - run: yarn install --frozen-lockfile
       - run: yarn e2e
-
-  # Join CI jobs into a single status so that only one status needs to be
-  # marked as required under branch protection rules.
-  all_ci_tests:
-    runs-on: ubuntu-latest
-    needs:
-      - test
-      - e2e
-    if: ${{ always() }}
-    steps:
-      - uses: cgrindel/gha_join_jobs@v1
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I had to remove this because for PRs from forks where we skip e2es, the joined check will fail when one of the inputs is skipped.